phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -1291,6 +1470,38 @@ public boolean next(List<Cell> results) throws IOException {
             } finally {
                 region.closeRegionOperation();
             }
+            if (verify) {
+                ArrayList<KeyRange> keys = new ArrayList<>(rowCountPerTask);
+                for (byte[] key : dataPutMap.keySet()) {
+                    keys.add(PVarbinary.INSTANCE.getKeyRange(key));
```

Comment:
Is there a difference between KeyRange.getKeyRange() and PDataType.getKeyRange() as you're using here?

Reply:
They are the same thing except that PDataType.getKeyRange() is a wrapper method of KeyRange.getKeyRange() and adjusts the key range to "
* Force lower bound to be inclusive for fixed width keys because it makes comparisons less expensive when you
* can count on one bound or the other being inclusive."